backend/src/ConnectionPool.ts

```diff
@@ -190,4 +191,9 @@ export default class ConnectionPool {
       return result
     }, [])
   }
+
+  openCMDforWindows(launchApp: string) {
+    Logger.error(launchApp)
+    child_process.exec(`start cmd.exe /K ${launchApp}`)
+  }
```

Comment:
Let's make a new file for this feature. Maybe call it `launch.ts`. I don't think it belongs in the connection pool class.